src/Core/Types/Unifier.cs

```diff
@@ -237,6 +237,25 @@ private bool IsCompatibleWithMemberPointer(MemberPointer mpA, DataType b, int de
             return false;
 		}
 
+        private bool CanBeMergedInto(DataType a, DataType b)
```

Comment:
Let's add a comment here like: `Returns true if the data type a can be unified into data type b`

Reply:
Done in 4626a2b.